docs/calico-with-docker/docker-network-plugin/README.md

```diff
@@ -235,7 +235,23 @@ fail.
 To see the list of networks, use
 
     docker network ls
-        
+
+## Assign Static IP when Starting a Container
+
+With the release of Docker 1.10, support has been added to allow users to
+configure a specific IP address when creating a container.  In order to use
+this feature, you must specify the `--subnet` flag when running `docker network
+create`.  This flag can be used in any of the four instruction sets below when
+creating the Docker network.
+
+If you are using the Calico IPAM driver, the `--subnet` value must fall within
+an existing Calico IP pool.  For example, if you create a Calico IP pool for
+`192.168.0.0/16`, you can use `--subnet=192.168.1.0/24` as a valid subnet.
+
+With this network, you can specify a given IP address when creating a
```

Comment:
Scans a bit funny.

Maybe change to:

For example, the following commands create a Calico network using a specific IP pool, and creates a container using a specific IP address from that pool:

and then incude the calicoctl pool command too.